README.md

```diff
@@ -74,11 +74,11 @@ fortio.exe server
 ```
 (at the prompt, allow the windows firewall to let connections in)
 
-Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/)
+Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio](http://localhost:8080/fortio).
```

Comment:
there is a reason for the trailing / - if you try you'll find out that without the trailing / you get one extra unnecessary redirect

Reply:
oh I see, sorry. Thanks for the info!